static/app/views/replays/detail/page.tsx

```diff
@@ -71,15 +60,7 @@ function Page({
             id: replayRecord.user.id || '',
           }}
           // this is the subheading for the avatar, so displayEmail in this case is a misnomer
-          displayEmail={
-            <Cols>
-              {frames?.length ? (
-                <FrameWalker replayRecord={replayRecord} frames={frames} />
-              ) : (
-                <StringWalker urls={replayRecord.urls} />
-              )}
-            </Cols>
-          }
+          displayEmail={<Cols>{undefined}</Cols>}
```

Comment:
i'm guessing this doesn't break 🤞 

```suggestion
          displayEmail={undefined}
```

Reply:
so removing `Cols` displays the email, it's weird
<img width="730" alt="SCR-20231003-mwsv" src="https://github.com/getsentry/sentry/assets/56095982/ad40f3dc-8c73-421f-a330-2837cc03b9a4">

but when `Cols` is there, no email

Reply:
ahh. thanks for checking.

can we put in a `<div/>` instead, then we don't need the extra `styled` call